docs/user_docs/spec_file_format.md

```diff
@@ -0,0 +1,35 @@
+# Spec file format
+**Will be rewritten** 
+This product uses CSV for file reading, this means that group definition is separated by a  ",". The spec for this product will have the following group definitions: name, ring quadrant, isNew, moved and description. For the CSV, this will look like:
+```
+name,ring,quadrant,isNew,moved,description
+```
+The reader will then translate these to structs, and therefor these groups will also have types.
```

Comment:
I propose we change this to
"The reader will then translate these to structs which get translated to types using Unmarshalling"